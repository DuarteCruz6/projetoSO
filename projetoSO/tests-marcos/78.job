SHOW
WRITE [(g6,78)(r0,61)(b3,81)(m5,87)]
WRITE [(w8,28)(w9,72)]
BACKUP
WRITE [(r9,55)(c5,29)(n5,1)]
DELETE [r0,m5,c5]
WRITE [(j7,23)(p4,94)(z5,49)(v5,55)(e7,93)]
BACKUP
WRITE [(w7,71)(m9,22)(m6,39)(o1,15)]
WRITE [(j3,10)]
WRITE [(k3,2)(k9,76)(t4,54)(i9,17)]
SHOW
READ [r9]
