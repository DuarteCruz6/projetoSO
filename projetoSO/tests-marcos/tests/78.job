BACKUP
WRITE [(q5,5)(a3,13)]
WRITE [(u4,74)(q4,44)(q4,16)(x3,42)]
WRITE [(f5,53)]
WRITE [(v7,81)(v7,89)]
READ [a3]
WRITE [(k4,32)(q1,22)]
WRITE [(g5,61)]
WRITE [(i0,51)(u0,90)(k6,57)(q6,52)(n4,49)]
WRITE [(m5,62)]
DELETE [g5,m5,u4,k6]
READ [a3]
WRITE [(c8,79)]
SHOW
BACKUP
WRITE [(t1,58)(f2,65)(d0,92)(e0,48)(a2,93)]
DELETE [v7,q5,q1,i0]
WRITE [(z8,99)]
WRITE [(v7,79)(j1,71)(g4,55)(w7,54)(o8,0)]
WRITE [(n6,75)(p3,82)(x0,26)]
WRITE [(i0,51)(h9,96)(x1,79)(t2,24)]
WRITE [(c4,42)(l6,53)(s3,69)(v0,53)]
READ [q4]
DELETE [x0,f2,t2]
READ [p3]
DELETE [f5,x1,n4]
WRITE [(g9,82)(k0,81)]
WAIT 2000
DELETE [q4,q6,a2,l6]
WRITE [(v5,1)]
WRITE [(h3,88)(e2,84)(j4,21)]
READ [j4]
READ [j1]
DELETE [v0]
DELETE [e0,e2,k4,x3]
WRITE [(r0,42)(c0,5)]
WRITE [(x4,16)(x4,86)(v6,53)]
WRITE [(w9,43)(w8,76)(e1,83)]
DELETE [c4,a3,o8]
WRITE [(u9,51)(f4,34)(o6,67)]
WRITE [(i5,85)]
WRITE [(t3,47)(f8,25)(q1,41)(z9,86)]
BACKUP
WRITE [(i9,44)(b8,63)(t2,93)]
WRITE [(g8,6)]
BACKUP
WAIT 1000
WAIT 3000
WRITE [(t0,38)(n0,91)(u5,100)(n8,90)(p6,57)]
WRITE [(p2,28)(h6,16)]
READ [c0]
WRITE [(b3,62)(c6,51)]
BACKUP
BACKUP
WRITE [(g3,48)(q0,76)(o7,14)(h0,55)]
DELETE [q1,v6,o6,z8,z9]
WRITE [(v8,36)(o5,59)(l2,40)]
READ [p3]
READ [t2]
WRITE [(u8,78)(i4,83)]
WRITE [(s1,96)(q7,37)(x5,97)]
DELETE [e1]
DELETE [u5,w8,h3,n8,i0]
BACKUP
DELETE [w9,g4,i9]
READ [p3]
WRITE [(l4,69)(n5,22)(x8,55)]
READ [l2]
WRITE [(z2,2)(o3,81)]
WRITE [(k3,25)(c1,96)(b0,1)(r2,66)]
WRITE [(i1,9)(a6,21)(j7,5)]
DELETE [p2,j4,q7,u0,h6]
WRITE [(c7,88)(m4,53)(f9,15)]
WAIT 2000
WRITE [(v6,42)(v2,35)(p7,64)]
READ [x8]
WRITE [(b4,25)(e0,45)]
WRITE [(a3,39)(l6,15)(q5,52)(w3,42)]
WRITE [(d9,40)(o9,64)(m8,68)]
DELETE [i4,h9,o7,v7]
DELETE [q0]
WRITE [(g4,71)]
READ [v8]
