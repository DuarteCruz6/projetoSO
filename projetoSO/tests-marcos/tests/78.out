[(a3,13)]
[(a3,13)]
(a3, 13)
(c8, 79)
(f5, 53)
(i0, 51)
(k4, 32)
(n4, 49)
(q6, 52)
(q1, 22)
(q4, 16)
(q5, 5)
(u0, 90)
(v7, 89)
(x3, 42)
[(q4,16)]
[(p3,82)]
[(j4,21)]
[(j1,71)]
[(c0,5)]
[(p3,82)]
[(t2,93)]
[(p3,82)]
[(l2,40)]
[(x8,55)]
[(v8,36)]
